src/theme/components/customTable.ts

```diff
@@ -1,8 +1,28 @@
+// What we might expect users to pass for the color "white". This is
+// temporary until we have a better color contrast checker.
+const whiteColors = [
+  "#fff",
+  "#ffffff",
+  "#FFF",
+  "#FFFFFF",
+  "white",
+  "var(--nypl-colors-ui-white)",
+];
 const baseStyle = ({
   columnHeadersTextColor,
   showRowDividers,
   useRowHeaders,
 }) => ({
+  // Just so we don't let row headers to display as white. Only the
+  // thead should have white text color with any color background.
+  tbody: {
+    th: {
+      color:
+        useRowHeaders && whiteColors.includes(columnHeadersTextColor)
+          ? "var(--nypl-colors-ui-black)"
+          : columnHeadersTextColor,
+    },
+  },
```

Comment:
<img width="1042" alt="Screen Shot 2022-02-15 at 5 12 53 PM" src="https://user-images.githubusercontent.com/30579702/154158504-7602b301-b4d4-4280-924f-bbee235cd962.png">

Reply:
@EdwinGuzman  I added **columnHeadersBackgroundColor** also "#FFF".   I am thinking  that both colors should not match.

Reply:
Hmmm that's a good point. I don't think anyone will purposely add white to both props... The initial issue that is fixed with this PR can happen by accident so it's more important. We can set this aside until we implement a color contrast checker or a better color validation repo-wide. I'm saying this as this issue also happens in other components such as the `Card`:

<img width="1190" alt="Screen Shot 2022-02-15 at 6 02 36 PM" src="https://user-images.githubusercontent.com/1280564/154164494-178d0083-7606-4de5-b5c1-977ff858e9ef.png">

So we can fix all of this in another PR.

Reply:
Ok Edwin